src/main/java/com/currencycloud/client/CurrencyCloudClient.java

```diff
@@ -53,11 +53,73 @@ public class CurrencyCloudClient {
     private String apiKey;
     private String authToken;
 
+    public static class HttpClientConfiguration {
+        private final int httpConnTimeout;
+        private final int httpReadTimeout;
+
+        private HttpClientConfiguration(int httpConnTimeout, int httpReadTimeout) {
+            this.httpConnTimeout = httpConnTimeout;
+            this.httpReadTimeout = httpReadTimeout;
+        }
+
+        public int getHttpConnTimeout() {
+            return httpConnTimeout;
+        }
+
+        public int getHttpReadTimeout() {
+            return httpReadTimeout;
+        }
+
+        public static class HttpClientConfigurationBuilder {
+            private int httpConnTimeout = 30000;
```

Comment:
@jonathancouchman the defaults are defined here for the configuration using the same default value as before and as defined here https://github.com/CurrencyCloud/currencycloud-java/blob/master/src/main/resources/rescu.properties